app/codecs/CirceDecoders.scala

```diff
@@ -119,6 +120,27 @@ object CirceDecoders {
   implicit val payPalErrorBodyDecoder: Decoder[PayPalError] = deriveDecoder
   implicit val payPalSuccessDecoder: Decoder[PayPalSuccess] = deriveDecoder
 
+  implicit val pricingDecoder: Decoder[Pricing] = deriveDecoder
```

Comment:
Circe will be able to do this with automatic derivation so the decoders and encoders shouldn't be needed. Happy to show you how.

Reply:
would love to know!